src/sql/workbench/services/connection/browser/cmsConnectionWidget.ts

```diff
@@ -108,8 +108,9 @@ export class CmsConnectionWidget extends ConnectionWidget {
 			} else {
 				authTypeOption.defaultValue = this.getAuthTypeDisplayName(AuthenticationType.SqlLogin);
 			}
-			this._authTypeSelectBox.setOptions(authTypeOption.categoryValues.map(c => c.displayName), 1);
 		}
+		this._authTypeSelectBox.setOptions(authTypeOption.categoryValues.map(c => c.displayName), 1);
+		this._authTypeSelectBox.selectWithOptionName(authTypeOption.defaultValue);
```

Comment:
Does this work as expected with registered servers? You're calling setOptions twice in that case.

Reply:
Yes, it defaults to the default value when cancelled.